### DISCUSSION TITLE
[Question][Hw1] How should we format the outputs?

### DISCUSSION THREAD

**Discussion by beomjk** [prosyslab-classroom/cs348-information-security#489] [Q&A]
I have some questions about the output format.

1. If the decrypted bytes are 0x4d49434845414c01 (for "MICHEAL" + single padding), is it okay to print "MICHEAL", or "MICHEAL\n" to stdout?
2. If 0x0808080808080808 is the one, that is, M is empty, then how to print it out? Just "\n" could be ambiguous with 0x2007070707070707 ...

**Comment by Hwi-gyeom** [comment]
1. Please make your output "MICHEAL"

2. There will be no empty plaintext in our assignment.

**Comment by adilb99** [comment]
@Hwi-gyeom , is it important to ensure then that no `\n` gets put with `print(plaintext, end='')` or just `print(plaintext)`  is fine? (Cause Python puts newline automatically when printing)

**Comment by Hwi-gyeom** [comment]
Reconsidering it, I think we need line change between 2 different plaintext.

Please make the print output like

MICHEAL
MIKE
...

**Comment by beomjk** [comment]
Do you mean `\n` should be added after the result? I'm confused about your output example since our work takes only C0 and C1 and prints single message as I know...

**Comment by KAIST-JongchanPark** [comment]
As adlib99 said above, when I print single line with python3 (e.g. print("something") ), "\n" is added automatically to end of "something". (i.e. "something\n" is printed.) 

Also, as i remember, the assignment required for printing only 1 plaintext.

If I print single line(i.e. plaintext) with "\n", could there be some problem?

**Comment by Hwi-gyeom** [comment]
I think anything will be okay.
